main.go

```diff
@@ -28,14 +31,35 @@ var (
 	setBasicAuth             = flag.String("set-basic-auth", "", "Define the basic auth. Form must be user:password")
 	defaultUsernameBasicAuth = flag.String("default-user-basic-auth", "gopher", "Define the user")
 	sizeRandom               = flag.Int("password-length", 16, "Size of the randomized password")
-	logRequest               = flag.Bool("enable-logging", false, "Enable log request")
```

Comment:
Most users are pulling the latest version of `goStatic`, so this will break deployments. I admit that `enable-logging` is not useful thanks to your PR. Maybe we could have both:

* `log-level` as you defined it
* `enable-logging` that will print a deprecated warning, then will force the debug level.

WDYT?

Reply:
Absolutely, commit has been added addressing that issue, let me know if that works.